Sources/Jobs/JobQueueHandler.swift

```diff
@@ -68,6 +69,14 @@ final class JobQueueHandler<Queue: JobQueueDriver>: Service {
         var logger = logger
         let startTime = DispatchTime.now().uptimeNanoseconds
 
+        let jobQueuedDuration = Date().timeIntervalSince(queuedJob.queuedAt)
+
+        // Calculate wait time from queued to processing
+        Timer(
+            label: "\(self.metricsLabel)_queued_for_duration_seconds",
+            preferredDisplayUnit: .seconds
+        ).recordNanoseconds(Int64(jobQueuedDuration * 100_000))
```

Comment:
This doesn't seem like a correct multiplication. Isn't a second `1_000_000` nanoseconds?

Reply:
Yep, that was a typo. Thanks!

Reply:
A second is 1_000_000_000 nanoseconds.

Reply:
You're right, 9 digits. Thanks for the correction